src/main/resources/templates/profile.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="https://www.thymeleaf.org/">
+<head th:replace="fragments :: head">
+    <meta charset="UTF-8">
+</head>
+    <body>
+    <nav th:replace="fragments :: header"></nav>
+<h1>User Profile</h1>
+    <hr>
+    <div class="media">
+        <img class="align-self-start mr-3" th:src="@{${user.profilePicture}}" alt="Profile Picture Image">
+        <div class="media-body">
+            <h5 class="mt-0">Welcome Back</h5>
+
+
+            <ul class="list-group">
+                <li class="list-group-item list-group-item-success" th:text=" 'Name: ' + ${user.firstName}"></li>
```

Comment:
Might want to change this list-group-item formatting so the box fits on the page. It currently bleeds off on the right side. This is just a suggestion for improving aesthetics.